Login_Logout.java

```diff
@@ -0,0 +1,98 @@
+package Lession3SeleniumWebPackage;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * Login/logout với tài khoản đã tạo
+ * 
+ * @author ThaoBTP
+ *
+ */
+
+public class Login_Logout {
+	public static WebDriver driver = null;
+	public static String loginPage = "http://live.guru99.com/index.php/customer/account/login/";
+	public static String accountPage = "http://live.guru99.com/index.php/customer/account/";
+
+	public static String expectedPageTitleMessage = "My Dashboard";
+	public static String validEmail = "name@gmail.com";
+	public static String invalidEmail = "123@abc.com";
+	public static String passWord = "123456";
+
+	public static String emailLocator = "id=email";
+	public static String passLocator = "id=pass";
+	public static String loginBtnLocator = "id=send2";
+	public static String accountLocator = "link=ACCOUNT";
+	public static String logoutLocator = "xpath=.//*[@title='Log Out']";
+	public static String pageTitleLocator = "xpath=.//*[@class='page-title']";
+	public static String errorMessageLocator = "xpath=.//*[@class='error-msg']";
+
+	// Login vào hệ thống
+	public static void login(String email, String passWord) {
+		CommonBase common = new CommonBase(driver);
+		try {
+			common.sendKeys(emailLocator, email);
+			common.sendKeys(passLocator, passWord);
+			// click login
+			common.click(loginBtnLocator);
+		} catch (Exception e) {
+			common.printLn(e.toString());
+		}
+	}
+
+	// Logout khỏi hệ thống
+	public static void logout() {
+		CommonBase common = new CommonBase(driver);
+		try {
+			common.click(accountLocator);
+			common.click(logoutLocator);
+		} catch (Exception e) {
+			common.printLn(e.toString());
+		}
+	}
+
+	public static void main(String[] args) {
```

Comment:
temporarily, you should create 2 functions to test 2 case: one success and one fail. Then in main function, call 2 functions, should not write code in main function as now